src/SharedUtilities/Drivers.jl

```diff
@@ -191,6 +191,12 @@ function surface_fluxes_at_a_point(
     h::FT = atmos.h
 
     ts_in = construct_atmos_ts(atmos, t, thermo_params)
+    q::FT = atmos.q(t)
```

Comment:
I think we keep dew/frost as an option?

Reply:
CLM seems to - Eqn 5.147, 5.148 https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf

Reply:
I think this is fine? But frost/dew/evap/sub all change SWE. the liquid fraction is still computed by a partitioning at T=T_f. it's not like frost change ice and dew changes liquid content directly.